.github/workflows/lib-injection.yml

```diff
@@ -5,14 +5,12 @@ on:
 
 jobs:
   build-and-publish-test-image:
-    uses: ./.github/workflows/build-and-publish-image.yml
-    with:
-      tags: 'ghcr.io/datadog/dd-trace-py/dd-lib-python-init:${{ github.sha }}'
-      platforms: 'linux/amd64,linux/arm64/v8'
-      build-args: 'DDTRACE_PYTHON_VERSION=git+https://github.com/Datadog/dd-trace-py@${{ github.sha }}'
-      context: ./lib-injection
+    uses: ./.github/workflows/lib-inject-publish.yml
     secrets:
       token: ${{ secrets.GITHUB_TOKEN }}
+    with:
+      ddtrace-version: v1.10
```

Comment:
Will we need to bump this tag after every release? Is there a way to automate this (ex: use latest)

Reply:
These tests ensure that the high-level strategy of using `dl_wheels.py` to get the package, `sitecustomize.py` to instrument works, using the current commit of the code would be ideal but would follow a different code path than the production code would.

There is a gap where the auto instrumentation step could fail but that falls in the domain of our integration tests for auto instrumentation.